twitter/endpoint.bal

```diff
@@ -15,25 +15,33 @@
 // under the License.
 
 import ballerina/http;
+import ballerina/time;
 import ballerina/url;
+import ballerina/uuid;
 
-# Twitter Client object.
+# Ballerina Twitter connector provides the capability to access Twitter API.
+# This connector lets you to perform operations related to Tweets and users.
 #
 # + apiKey - Consumer key of the Twitter account
 # + apiSecret - Consumer secret of the Twitter account
 # + accessToken - Access token of the Twitter account
 # + accessTokenSecret - Access token secret of the Twitter account
-# + twitterClient - HTTP Client endpoint
+# + twitterClient - Connector HTTP endpoint
 @display {label: "Twitter"}
-public client class  Client {
+public isolated client class  Client {
     
-    string apiKey;
-    string apiSecret;
-    string accessToken;
-    string accessTokenSecret;
+    private final string apiKey;
+    private final string apiSecret;
+    private final string accessToken;
+    private final string accessTokenSecret;
 
-    http:Client twitterClient;
+    private final http:Client twitterClient;
 
+    # Initializes the connector. During initialization you have to pass API credentials.
```

Comment:
To initialize, you have to pass API credentials.

Reply:
Corrected doc